test/image/mocks/line_scatter.json

```diff
@@ -30,7 +30,8 @@
         9
       ],
       "mode": "lines",
-      "type": "scatter"
+      "type": "scatter",
+      "opacity": 0.5
```

Comment:
Would be a better test to set the opacity on a lines+markers trace - which I don't think is handled correctly in this PR, trace opacity should get set on `traceGroup` rather than independently to lines and markers (and fills!).